python/Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
 name = "cql2-python"
-version = "0.2.0"
-authors = ["David Bitner <bitner@dbspatial.com>"]
-edition = "2021"
+version = { workspace = true }
+authors = { workspace = true }
+edition = { workspace = true }
 description = "Parse, validate, and convert Common Query Language (CQL2) text and JSON"
 readme = "README.md"
 homepage = "https://github.com/developmentseed/cql2-rs"
```

Comment:
You can probably DRY the homepage too if you want

Reply:
Thought about that, but then realized I probably want to correct that later (once the docs are published) to make the Python homepage https://developmentseed.org/cql2-rs/latest/python/ (and probably do the same for the CLI as well).